src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RGBA.txt

```diff
@@ -0,0 +1,41 @@
+﻿>> RGBA(1, 2, 3, 0.015)
+Color [A=4, R=1, G=2, B=3]
+
+>> RGBA(0, 0, 0, 0)
+Color [A=0, R=0, G=0, B=0]
+
+>> RGBA(255, 255, 255, 1.0)
+Color [A=255, R=255, G=255, B=255]
+
+>> RGBA(-1, 255, 255, 1.0)
+#Error(Kind=InvalidArgument)
+
+>> RGBA(1.5, 255, 255, 1.0)
```

Comment:
We have many functions that expect integer arguments (Left, Mid, Char, Replace) and are ok with fractional arguments... In those cases we truncate them, so this one should be equivalent to RGBA(1,255,255,1)

Reply:
I changed to truncating. Still need to consider lower and upper bounds...

Reply:
Talking to Greg it sounds we all agree this should not allow out-of-bound values like -1 or 256 as silently clamping is not obvious. We'll need to consider what to do on Power Apps for backward compatibility.